app/models/game.rb

```diff
@@ -1,4 +1,5 @@
 class Game < ActiveRecord::Base
   validates :name, presence: true
+  scope :available, -> {where(white_player_id == nil || black_player_id == nil)}
```

Comment:
Try writing a test for this.

We don't have factories setup yet. So this could be a good opportunity to set that up for games/users.

To get you started, this is what the test might look like (you'd have to setup the Factory for game/user for this to work)
```Ruby
it 'returns available games' do
   available_game = create(:game, white_player: user, black_player: nil)
   expect(Game.available).to include available_game
end
```